Amasty_GiftCard/templates/checkout/cart/check.phtml

```diff
@@ -4,27 +4,27 @@
 $card = $block->getCard();
 $currentBalance = $block->getCurrentBalance($card->getCurrentValue());
 if ($card->getExpiredDate()) {
-    $_validTill = $block->getExpiredDate($card->getExpiredDate());
+    $_validTill = /* @noEscape */ $block->getExpiredDate($card->getExpiredDate());
```

Comment:
we don't add comments when assigning variables. You can move this where there is echo statemnt

Reply:
I don't see here required changes. But I know that you didn't get here yet. But to be clear.

When we add `/* noEscape */` or '$block->escapeHtml()` or other method or "nothing". 
We add it when we use echo statements. Not during assigning value. So here we should remove noEscape.
Wrong:
```
$someValue = /* @noEscape */ $block->getSomeValue();
```
Good:
```
<?php 
$someValue = $block->getSomeValue();
?>
....
<?= /* @noEscape */ $someValue ?>
```